api/db/create_db.sql

```diff
@@ -1,16 +1,20 @@
 CREATE DATABASE IF NOT EXISTS ${DB_NAME} CHARACTER SET utf8 COLLATE utf8_general_ci;
 
+CREATE USER IF NOT EXISTS '${DEV_READONLY_DB_USER}'@'%' IDENTIFIED BY '${DEV_READONLY_DB_PASSWORD}';
```

Comment:
nit: I think this would be a bit more readable if the CREATE/PASSWORD/GRANT statements were grouped by user instead of by statement type.

Reply:
I'd rather keep all the user initialization first before executing any other GRANT statements